xmppserver/src/main/java/org/jivesoftware/openfire/group/DefaultGroupPropertyMap.java

```diff
@@ -32,7 +40,7 @@
 
 public class DefaultGroupPropertyMap<K,V> extends PersistableMap<K,V> {
 
-    private static final long serialVersionUID = 3128889631577167040L;
+    private static final long serialVersionUID = -7501088917416583815L;
```

Comment:
Could this change in serialVersionUID cause problems in mixed-version clusters, or during upgrade procedures?

Reply:
I'm not sure, but I don't think that the change is needed at all. The composition of the object did not change. I'll revert this.